convert.go

```diff
@@ -50,21 +51,27 @@ func IsFloat64AJSONInteger(f float64) bool {
 	return diff/math.Min(fa+ga, math.MaxFloat64) < epsilon
 }
 
-var evaluatesAsTrue = map[string]struct{}{
-	"true":     {},
-	"1":        {},
-	"yes":      {},
-	"ok":       {},
-	"y":        {},
-	"on":       {},
-	"selected": {},
-	"checked":  {},
-	"t":        {},
-	"enabled":  {},
+var atomicEvaluatesAsTrue atomic.Value
+
+func init() {
+	atomicEvaluatesAsTrue.Store(
+		map[string]struct{}{
+			"true":     {},
+			"1":        {},
+			"yes":      {},
+			"ok":       {},
+			"y":        {},
+			"on":       {},
+			"selected": {},
+			"checked":  {},
+			"t":        {},
+			"enabled":  {},
+		})
 }
 
 // ConvertBool turn a string into a boolean
 func ConvertBool(str string) (bool, error) {
+	evaluatesAsTrue := atomicEvaluatesAsTrue.Load().(map[string]struct{})
```

Comment:
@fredbi What is the point of this atomic?

Reply:
Concurrent reads on such a global map are not safe.
I had two solutions in mind: either use sync.Map or an atomic. I preferred the latter a non-locking (it is a static configuration)

Reply:
concurrent reads are not fine
regarding your other remark, you might be right - I have to check that further

Reply:
Yes, concurrent reads are fine.

https://golang.org/doc/faq#atomic_maps

>Map access is unsafe only when updates are occurring. As long as all goroutines are only reading—looking up elements in the map, including iterating through it using a for range loop—and not changing the map by assigning to elements or doing deletions, it is safe for them to access the map concurrently without synchronization.

Reply:
ok you got  a good point. Thank you. I am going to revert this

Reply:
You're welcome.